### PR TITLE
did tweaks in all the test classes

### PR DESCRIPTION
**Pull request by debragordon**
@drdreed @GAylabea @abidina updated the test files ... the parser ones are still failing ... we need to fix/resolve them by tweaking the parser class

### CODE REVIEW

File: SimpleCalculator.Tests/EvaluateTests.cs [hunk 1]
```diff
@@ -8,13 +8,20 @@ public class EvaluateTests
     {
         Evaluate TestEvaluate = new Evaluate();
```
Comment (DevDreed) [inline SimpleCalculator.Tests/EvaluateTests.cs]: This line should be moved to the first test.  All tests need to start with a clean base.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Made comments inline

**Comment by debragordon** [comment]
@drdreed i respushed this file (didn't scroll down to see this^^) sorry ... does this new file work then?

**Review by DevDreed — APPROVED** [review]
👍

**Comment by debragordon** [comment]
string pattern = @"([a-zA-Z]{1}|\d+)\s*([\+\-*\/%\=]{1})\s*([a-zA-Z]{1}|\d+)";

**Pull request merged**